docs/env-variables.md

```diff
@@ -0,0 +1,27 @@
+# Different Environment Variables used
+
+This page describes about the different environment variables that are
+used to configure kube-scheduler-simulator.
+
+`Port`: This is the port number on which kube-scheduler-simulator is started and it's value is 1212 which is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L7)
+
+`EtcdURL`: This is the URL for kube-scheduler-simulator etcd which runs locally and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L8)
+
+`FrontendURL`: This URL is on which kube-scheduler-simulator is reachable locally and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/3111ec419ccb8c49197b75385fbd166f7f159435/docker-compose.yml#L9).
+
+`KUBECONFIG`: Currently not used. [Beta: Existing Cluster Importing]
+This variable is used to find the information it needs to access the
+cluster which will be imported to our simulator and communicate with
+the API server of the cluster. If you use the Beta feature, you need
+to set the `Kubeconfig` file path of the cluster which will be
+imported. [Read more]. [Read
+more](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#the-kubeconfig-environment-variable)
+
+`KUBE_API_HOST`: This is the host on which the kube-apiserver serves
+for the simulator server.
+
+`KUBE_API_SERVER_URL`: This is the kube-apiserver URL and it's value is set [here](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/docker-compose.yml#L26)
```

Comment:
This `KUBE_API_SERVER_URL` is env for frontend. And I think we should write the doc about env for frontend in another section to avoid confusion.

Please create another section (like `## For web UI`) and move `KUBE_API_SERVER_URL` in the section

Reply:
Done, thanks for mentioning that.